pytest_zigzag/__init__.py

```diff
@@ -152,14 +148,15 @@ def pytest_sessionfinish(session):
     SESSION_MESSAGES.drain()  # need to reset this on every pass through this hook
     if session.config.pluginmanager.hasplugin('junitxml'):
         zz_option = _get_option_of_highest_precedence(session.config, 'zigzag')
-        qtest_project_id = _get_option_of_highest_precedence(session.config, 'qtest-project-id')
-        if zz_option and qtest_project_id:
+        pytest_zigzag_config = _get_option_of_highest_precedence(session.config, 'pytest-zigzag-config')
+        if zz_option and pytest_zigzag_config:
             try:
                 junit_file_path = getattr(session.config, '_xml', None).logfile
+
                 # noinspection PyTypeChecker
                 # validate token
                 token = _validate_qtest_token(os.environ['QTEST_API_TOKEN'])
-                zz = ZigZag(junit_file_path, token, qtest_project_id, None)
+                zz = ZigZag(junit_file_path, pytest_zigzag_config, token)
```

Comment:
This assumes that the `pytest_zigzag_config` file contains both "pytest-zigzag" and "zigzag" configuration settings which is not guaranteed. The documentation should be updated to reflect the best practice for invoking "zigzag" with the right configuration settings.